launcher_scripts/nemo_launcher/core/v2/config_k8s.py

```diff
@@ -179,6 +179,7 @@ class K8sClusterConfig(BaseModel):
     capabilities: Optional[list[str]] = [
         "IPC_LOCK"
     ]  # capabilities to add to all containers (useful for debugging), ex. ["IPC_LOCK", "SYS_PTRACE"]
+    scheduler: Optional[str] = "null"  # Specify a custom kubernetes scheduler if different from default, such as "runai-scheduler". Leave as "null" if the default scheduler is desired.
```

Comment:
You should just be able to leave this as the following:
```suggestion
    scheduler: Optional[str] = None  # Specify a custom kubernetes scheduler if different from default, such as "runai-scheduler". Leave as "null" if the default scheduler is desired.
```

There's `core.v2.step_k8s.prune_tree` which will prune that key if it's in the manifest as `None`

Reply:
Ah, didn't realize that, thanks @terrykong! I'll make that change.